data/live-build-config/includes.chroot/usr/lib/live/boot/9991-vyos.sh

```
@@ -0,0 +1,316 @@
+	if [ ! -b "${device}" ]
+	then
+		return 1
+	fi
+
+	# VyOS: never mount a member of an assembled MD RAID directly - doing so
+	# would give us a second, inconsistent view of the same filesystem.
+	raid_drives=$(awk '{ if ($4 != "name") { print $4 } }' /proc/partitions \
+			| grep "^md" || true)
+	for raid_drive in ${raid_drives}
+	do
+		disks=$(ls /sys/block/${raid_drive}/slaves 2>/dev/null || true)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)